### ISSUE TITLE
[1.6-rc.15] Unable to create projects / run `meteor npm i` 

### ISSUE THREAD

**Issue by nathantreid** [meteor/meteor#9247]
# Environment
OS: Windows 10 x64
Meteor: 1.6-rc.15 (last version of Meteor with which an install succeeds without errors is 1.5.2.2)

# Expected behavior
Running `meteor create` or `meteor npm i` should succeed.

# Actual behavior
Not sure if this is related to #9243. As noted in [a comment there](https://github.com/meteor/meteor/issues/9243#issuecomment-337740874), `npm i` does work, after which the app can be run using the `meteor` command.

The first `meteor create` hung on installing npm packages:
``` bat
C:\scratch>meteor create --release 1.6-rc.15 1.6rc15
Terminate batch job (Y/N)? ^Cs               \
```

The second, third, etc `meteor create`s failed with output:
``` bat
C:\scratch>meteor create --release 1.6-rc.15 1.6rc15a
H:\.meteor\packages\meteor-tool\1.6.0-rc.15\mt-os.windows.x86_32\dev_bundle\lib\node_modules\meteor-promise\promise_server.js:218
      throw error;
      ^

Error: Error: Could not install npm dependencies for test-packages: Command failed: C:\WINDOWS\system32\cmd.exe /c H:\.meteor\packages\meteor-tool\1.6.0-rc.15\mt-os.windows.x86_32\dev_bundle\bin\npm.cmd install
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\cmd\git.EXE ls-remote -h -t https://github.com/meteor/readable-stream.git
npm ERR!
npm ERR! fatal: Unable to find remote helper for 'https'
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\nathantreid\AppData\Roaming\npm-cache\_logs\2017-10-19T17_30_36_807Z-debug.log
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\cmd\git.EXE ls-remote -h -t https://github.com/meteor/readable-stream.git
npm ERR!
npm ERR! fatal: Unable to find remote helper for 'https'
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\nathantreid\AppData\Roaming\npm-cache\_logs\2017-10-19T17_30_36_807Z-debug.log

    at Object.error (H:\tools\utils\buildmessage.js:430:11)
    at H:\tools\cli\default-npm-deps.js:32:20
    at H:\tools\utils\buildmessage.js:334:18
    at exports.EnvironmentVariable.withValue (H:\tools\utils\fiber-helpers.js:89:14)
    at H:\tools\utils\buildmessage.js:333:36
    at exports.EnvironmentVariable.withValue (H:\tools\utils\fiber-helpers.js:89:14)
    at Object.enterJob (H:\tools\utils\buildmessage.js:324:26)
    at Object.install (H:\tools\cli\default-npm-deps.js:27:27)
    at Command.func (H:\tools\cli\commands.js:799:36)
    at H:\tools\cli\main.js:1522:15
```

Referenced log file (1379 lines):
[2017-10-19T17_30_36_807Z-debug.log](https://github.com/meteor/meteor/files/1399465/2017-10-19T17_30_36_807Z-debug.log)

`meteor npm i` fails with the same error:
``` bat
C:\scratch\1.6rc15a>meteor npm i
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\randombytes-05526b9a\package.json'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\package.json'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\diffie-hellman-ddc56d51\package.json'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\browserify-rsa-d8e3e3e7\package.json'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\randombytes-05526b9a\.npmignore'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\.npmignore'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\diffie-hellman-ddc56d51\.npmignore'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\browserify-rsa-d8e3e3e7\LICENSE'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\randombytes-05526b9a\README.md'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\README.md'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\diffie-hellman-ddc56d51\browser.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\browserify-rsa-d8e3e3e7\index.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\randombytes-05526b9a\LICENSE'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\LICENSE'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\diffie-hellman-ddc56d51\index.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\browserify-rsa-d8e3e3e7\test.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\randombytes-05526b9a\index.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\index.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\diffie-hellman-ddc56d51\.travis.yml'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\browserify-rsa-d8e3e3e7\.travis.yml'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\randombytes-05526b9a\browser.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\hash.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\browserify-rsa-d8e3e3e7\readme.md'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\http-browserify-3d49346b\example\streaming\main.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\randombytes-05526b9a\test.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\bin.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\vm-browserify-f7370a14\example\run\bundle.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\http-browserify-3d49346b\example\streaming\server.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\randombytes-05526b9a\.zuul.yml'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\sha.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\vm-browserify-f7370a14\example\run\entry.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\http-browserify-3d49346b\example\streaming\index.html'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\randombytes-05526b9a\.travis.yml'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\sha1.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\vm-browserify-f7370a14\example\run\server.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\elliptic-e0fa72de\lib\elliptic\ec\index.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\sha224.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\vm-browserify-f7370a14\example\run\index.html'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\elliptic-e0fa72de\lib\elliptic\ec\key.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\sha256.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\vm-browserify-f7370a14\readme.markdown'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\elliptic-e0fa72de\lib\elliptic\ec\signature.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\sha384.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\sha512.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70\.travis.yml'
npm WARN tar ENOENT: no such file or directory, lstat 'C:\scratch\1.6rc15a\node_modules\.staging\elliptic-e0fa72de\lib\elliptic'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\buffer-b7dbc82c\test\node\test-buffer-alloc.js'
npm WARN tar ENOENT: no such file or directory, lstat 'C:\scratch\1.6rc15a\node_modules\.staging\meteor-node-stubs-cef70e8b\node_modules\elliptic\lib\elliptic\eddsa'
npm WARN tar ENOENT: no such file or directory, lstat 'C:\scratch\1.6rc15a\node_modules\.staging\meteor-node-stubs-cef70e8b\node_modules'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\meteor-node-stubs-cef70e8b\node_modules\vm-browserify\example\run\bundle.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\meteor-node-stubs-cef70e8b\node_modules\vm-browserify\example\run\entry.js'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\meteor-node-stubs-cef70e8b\node_modules\vm-browserify\example\run\index.html'
npm WARN tar ENOENT: no such file or directory, open 'C:\scratch\1.6rc15a\node_modules\.staging\meteor-node-stubs-cef70e8b\node_modules\vm-browserify\example\run\server.js'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: md5.js@1.3.4 (node_modules\meteor-node-stubs\node_modules\md5.js):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: Cannot cd into 'C:\scratch\1.6rc15a\node_modules\.staging\md5.js-c8bab818'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: evp_bytestokey@1.0.3 (node_modules\meteor-node-stubs\node_modules\evp_bytestokey):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: Cannot cd into 'C:\scratch\1.6rc15a\node_modules\.staging\evp_bytestokey-da301dff'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: diffie-hellman@5.0.2 (node_modules\meteor-node-stubs\node_modules\diffie-hellman):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: Cannot cd into 'C:\scratch\1.6rc15a\node_modules\.staging\diffie-hellman-ddc56d51'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: vm-browserify@0.0.4 (node_modules\meteor-node-stubs\node_modules\vm-browserify):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: Cannot cd into 'C:\scratch\1.6rc15a\node_modules\.staging\vm-browserify-f7370a14'
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: sha.js@2.4.9 (node_modules\meteor-node-stubs\node_modules\sha.js):
npm WARN enoent SKIPPING OPTIONAL DEPENDENCY: ENOENT: Cannot cd into 'C:\scratch\1.6rc15a\node_modules\.staging\sha.js-46cbcd70'

npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\cmd\git.EXE ls-remote -h -t https://github.com/meteor/readable-stream.git
npm ERR!
npm ERR! fatal: Unable to find remote helper for 'https'
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     H:\.meteor\packages\meteor-tool\1.6.0-rc.15\mt-os.windows.x86_32\dev_bundle\.npm\_logs\2017-10-19T17_53_31_672Z-debug.log
```

Referenced log file:
[2017-10-19T17_53_31_672Z-debug.log](https://github.com/meteor/meteor/files/1399477/2017-10-19T17_53_31_672Z-debug.log)

**Comment by benjamn** [comment]
It sounds like something in your partially installed `node_modules` directory ended up corrupt. Have you tried `rmdir node_modules` followed by a fresh `meteor npm i`?

**Comment by nathantreid** [comment]
Yes, that results in the same error:
``` bat
C:\scratch\1.6rc15a>rmdir /s /q node_modules

C:\scratch\1.6rc15a>meteor npm i
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\cmd\git.EXE ls-remote -h -t https://github.com/meteor/readable-stream.git
npm ERR!
npm ERR! fatal: Unable to find remote helper for 'https'
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     H:\.meteor\packages\meteor-tool\1.6.0-rc.15\mt-os.windows.x86_32\dev_bundle\.npm\_logs\2017-10-19T19_59_45_597Z-debug.log
```

Also, sorry for not making it clear - the original install that hung was in a separate directory. I've tried `meteor create` multiple times - the first time (`meteor create --release 1.6-rc.15 1.6rc15`) it hung; the other times (`meteor create --release 1.6-rc.15 1.6rc15a`, `meteor create --release 1.6-rc.15 1.6rc15b`, etc) it failed with the error in the OP.

**Comment by benjamn** [comment]
Can you run that command successfully yourself?
```sh
git ls-remote -h -t https://github.com/meteor/readable-stream.git
```

**Comment by nathantreid** [comment]
Yes:
```
C:\scratch\1.6rc15a>git ls-remote -h -t https://github.com/meteor/readable-stream.git
1227c7b66deedb1dc5284a89425854d5f7ad9576        refs/heads/1.0.x
52550840cb1d6e8a98ef9a909a4bea360bc6f7da        refs/heads/1.1.x
b59a2b74b2350ab8ef948dca1c078ebda557ebad        refs/heads/add-mcollina
50c9386da190dfc1d646efb99e95f77621d2a4e4        refs/heads/backport-env-1-0
c03d159affdc6816c4e7d5ff42236ea460cc7242        refs/heads/backport-env-1-1
1cc3d27d351eabb3a28d4189dd47516cc8c56122        refs/heads/build-5.0.0
b95e3fb85e231aabf88f0bd9375cc402a6d0b493        refs/heads/build-5.1.0
ffc931b71242763bb07ba302e42fe2bc86fa8867        refs/heads/build-5.5.0
7e9bf6fbfbf4cbc3cef033a2438530c3d447622e        refs/heads/build-5.6.0
c9364ba21d0d1afd3f044e60141b1d480944d70b        refs/heads/build-5.7.0
7008a78b1580f93cd27e691adf486618a84cfde3        refs/heads/build-5.8.0
f47ee5c0a574be0fb1c915860a9666b1d9cc5989        refs/heads/build-5.9.1
fe2de8455803d5b7e25b7f8608a7de4acbcbeb8e        refs/heads/build-6.1.0
2e9f4ed76356b5651679ff5502de47417bbec423        refs/heads/build-6.2.1
7ca6e8e867fc996866d651db0a6b12fd240bb06e        refs/heads/build-6.3.1
3e8da336ec16f72be5f37c9857fb4fdb8db04944        refs/heads/build-6.4.0
d83d31c786fc6b33df9507d30669b42cbeece61b        refs/heads/build-6.5.0
80dc00cb05335c5fbadf50a81650cba295a0e3b5        refs/heads/build-7.0.0
0c6776faa92ee4122313973a7de992dcb61da174        refs/heads/build-7.3.0
8088b3705ff25a5203d9f2e1b0f208da32d7d12a        refs/heads/cleanup-readme
301294459ff78481c9818c936e71e5369d2f5aed        refs/heads/docs-fix
fd20cb5fdbdb8b53d415420374ac5b6b064e5c95        refs/heads/easier-stream-flag
ddf359568e75761eba5ad4892ea55846ef4d23b3        refs/heads/edge
fce4f68edce564caccd48aefbae10ee85c6ac9b4        refs/heads/es3
d0d888fc86cb30d33ca45f7adad3ac68e7a9afbe        refs/heads/fix-build
289786ac13c6f4b9c720278d0089c0a864656ea3        refs/heads/gulb-wierdness
69d13e3747f7c3596a6cbaf266f43f6d8e5fcb7d        refs/heads/inline-isarray
a34caf2fb38622b9694f44b1b93fbeeb1c9b016b        refs/heads/iojs-2.3.4
d64a64aa6061b9b6855feff4d09e58fb3b2e4502        refs/heads/master
91173c7289ad37914745c42e37573273885e21ec        refs/heads/merge-io-stream-back
0c7bc63cd7f87534e5f698d10bdb285661cd1165        refs/heads/multipipe
e9589f0624fb93025682512048f8b48ab26189fc        refs/heads/ngrok
66aa1ca5b635305cfc6fc518149be8034a78fe9c        refs/heads/no-isarray
da3c2e0f9c20cf6a458f679a380465960516a763        refs/heads/no-more-immediate
b655582b4a493a1f9d8ed786cb16f41af81ceaa1        refs/heads/nsimmons-do_not_overwrite_module_var
a14a47fa86fdd9f6969e439ff7ee9b1a2f5adf93        refs/heads/ofrobots-duplex-require
a16228b3d036836b11893a2ac0a51c459e4732ed        refs/heads/passthrough
82ae1536d350d2d3936b7f1ca2a8e4c1b8f2c217        refs/heads/read-cb
fe229674d6f2a764bd6b15f3b46145518b142101        refs/heads/remove-bundled-deps
e0dabb908d15a291e0460a09c2e9540937d84b78        refs/heads/rm-d
f8058dc49f8cff6453888f3b533e740cd38cb22d        refs/heads/rm-stream-cruft
1deb8cd2d74fe703e4911b839bcef3dec8a01885        refs/heads/safe-buffer
4102464299a5a5ba94307776863614042c9ece35        refs/heads/sync-0-10
b3cf98b93b93b2b7784b73c57612536463033f09        refs/heads/tests
c2d1cce2a5000507c18481f703d28fe51ad7aab2        refs/heads/update-2.5.0
add1bdbe1238ea225e585da5035bd262fb879edd        refs/heads/update-zuul
cd9bc3fbcdc82211c5649b5170706b10c65c8479        refs/heads/v0.10
1ea8b3c55aabf10dfee5cb0009e768d6967e6a94        refs/heads/zertosh-browserify-compat
7c066ed72f2354f7377f91459cae6de10f812416        refs/tags/v0.0.1
846f891a1ca394f729995d0a748a0e176c97e392        refs/tags/v0.0.1^{}
703b1d6a04cb4cdbd232f07307cb4356044c8e7b        refs/tags/v0.0.2
0c7bc63cd7f87534e5f698d10bdb285661cd1165        refs/tags/v0.0.2^{}
5023b6c8d149a36710ed6d9be13628306d4fb68c        refs/tags/v0.0.3
6fdbbf4b300da59a566911d10733794bf43b9aad        refs/tags/v0.0.3^{}
2deb9e84f5561d46a8a4729307d930b7e084d6d3        refs/tags/v0.0.4
ee0c8c20f320d5dca8d94b6f45eea89c50506234        refs/tags/v0.0.4^{}
64c876cd44c1854c50e08a2e6405c15a17b014b1        refs/tags/v0.1.0
5a395436ac170742221e227ea93c05350c538ccb        refs/tags/v0.1.0^{}
1c1bdb789017c435827c02489e4c1a341488bf1a        refs/tags/v0.2.0
bdc95b4a4a17bb2aae78d8f35bbf34cba1245dc1        refs/tags/v0.2.0^{}
128d624e67dd5e4b96f221ef40b544dbfeac77ef        refs/tags/v0.3.0
4b46fa1cddf642c4f8ad766100b871c6863675c4        refs/tags/v0.3.1
080a169e721b9c4752c9a1d67bd10815c83bf221        refs/tags/v1.0.0
d483fe0382f3d9037945ef9e351091c33d16dd82        refs/tags/v1.0.0^{}
bb0abe0703e527268811b3701a7056133ed857b6        refs/tags/v1.0.1
99a115d416d8c676739534b0f77406249c03db7f        refs/tags/v1.0.1^{}
46d07ccab8bdd4cf23848bb2313189a3ec5ae16c        refs/tags/v1.0.15
32b81f005472711b31e14cffcd591ab0715f0970        refs/tags/v1.0.15^{}
c5d6aa08eb3fa594ac6b8dbd553e921973cf182d        refs/tags/v1.0.17
dbcbd7c9e41cef5eb28029dc769dbbbd8eb24634        refs/tags/v1.0.17^{}
b129d2163705476cab671318bd81dee0e6a375a8        refs/tags/v1.0.2
8625e8fc8a06866c2e389f4ee97433f465e873aa        refs/tags/v1.0.2^{}
c4de4ca1df3cbdfb71b537ae6b591ff9da33c806        refs/tags/v1.0.24
7e0ae950d49ec44ac7074fdaf4c0989acf4f63bf        refs/tags/v1.0.24^{}
03ffdf82dc28d67a67659ac110290dcfc2186494        refs/tags/v1.0.25
8d0f2472304cffbe25d0eac81f633fe3679a0109        refs/tags/v1.0.25^{}
bb1d268d2ef1c395ce695869fd433de05466e7bd        refs/tags/v1.0.25-1
f22cd8c83f45f7b7641548f307191077c4fea1fc        refs/tags/v1.0.25-1^{}
9f9dc5f77c05ed848160e908bfc9e371bcddd349        refs/tags/v1.0.26
457c0c77c5a3f9b043084b169e0816fbfe20d038        refs/tags/v1.0.26^{}
a703ccb854688bd7c0e06b6519093f6ad4cdff4c        refs/tags/v1.0.26-1
d59a3d1d6c540645e9b049922199ad9bfe67e35f        refs/tags/v1.0.26-1^{}
29690be327bfb4f41a05a7ed10464d7d08920285        refs/tags/v1.0.26-2
3332e5a434bbc61878b7364576020c2f5835b39a        refs/tags/v1.0.26-2^{}
fa29c4bcbf115060ca0a5ebd5d7477b2cc330d79        refs/tags/v1.0.26-3
aaee41cd37f587ef55f56deaa9aa584e96e2a061        refs/tags/v1.0.26-3^{}
1f2441438ec2f54f3006b8ce0331b32f8e15a311        refs/tags/v1.0.26-4
36c101a8d2b12bdbe6fec4191ef7f087be66d420        refs/tags/v1.0.26-4^{}
ae8e99043217b449fd32cb5f6a02b15c5ef056dc        refs/tags/v1.0.27-1
c384448d1ec755895ae9356b91e9ee57cc00882c        refs/tags/v1.0.27-1^{}
a09f93a89413f011d7979f0ed80e72569010074d        refs/tags/v1.0.31
faf5d9def4bc1e4cf7f93015b59942b4d7b08e96        refs/tags/v1.0.31^{}
24288c7fe5b6381304044ef4350d760e40eb7968        refs/tags/v1.0.32
8ca1d1831525e69d1843f7acfaf4b7ed6566fdcc        refs/tags/v1.0.32^{}
2eacb1727533044d0bceb193613917f1fd8f0f3f        refs/tags/v1.0.32-1
ad328399d5ffb05f54455744d7c1a4848d1de5c0        refs/tags/v1.0.32-1^{}
98fc54f28f192ea92ad78ab8ebd6c1e5aab9859c        refs/tags/v1.0.33
0bf97a117c5646556548966409ebc57a6dda2638        refs/tags/v1.0.33^{}
b0048e6a91f2fdc8367b8b362317764fe97953e6        refs/tags/v1.0.33-1
cd9bc3fbcdc82211c5649b5170706b10c65c8479        refs/tags/v1.0.33-1^{}
6cae1faebcf2b01f05cfe39f390e1c9eb3996182        refs/tags/v1.0.33-2
99b64f21f49229b53f30f3fc5e9cffc4fcb84cb6        refs/tags/v1.0.33-2^{}
59a218f299c0f65f19ee8a7ec1c0990ed86087fc        refs/tags/v1.0.34
1227c7b66deedb1dc5284a89425854d5f7ad9576        refs/tags/v1.0.34^{}
ce2ea12caef74d616eddadf171a1ae11c743d4ba        refs/tags/v1.1.10
40fa7fc1bea7d7a6799e7ca1352551c4ad72e9bc        refs/tags/v1.1.10^{}
bf0d787b6c37b106b6b07c347e2f7b8289701dd8        refs/tags/v1.1.11
1d3690a2ae59cb2ce4ca8f8c36b0b840c9b47a8b        refs/tags/v1.1.11^{}
7fe266523e67f244415e13589fbac42bea664652        refs/tags/v1.1.11-1
116b26af87c43be65704da7338f16414ca4c5e09        refs/tags/v1.1.11-1^{}
3e279426d2b960f167eb886aca3adb80c2022437        refs/tags/v1.1.12
dd90019bd3cdf2b5634f7b559db374b70533dc65        refs/tags/v1.1.12^{}
45bf401402d64468c4e27e7924be3fc637e4b9b0        refs/tags/v1.1.12-1
dd1ba8ccf427a31897f907ef1f3484ce0c4c5131        refs/tags/v1.1.12-1^{}
a07763d4f1aca5c489a3dc3be8b207442fcf750b        refs/tags/v1.1.13
3b672fd7ae92acf5b4ffdbabf74b372a0a56b051        refs/tags/v1.1.13^{}
54e70aa5e53ce5712d28787825d3a9b700baedd3        refs/tags/v1.1.13-1
d18dd32fa6cf9d79314ee45c758ae8af18c84f51        refs/tags/v1.1.13-1^{}
59dea73963ab61377bd75f659ec28e8f7d283bac        refs/tags/v1.1.14
52550840cb1d6e8a98ef9a909a4bea360bc6f7da        refs/tags/v1.1.14^{}
a26a02ea97ad9b8c1c64453105c3489cd2799b64        refs/tags/v1.1.7
d97bc949bc2b66d49c4bdf4972ee7bea08370db6        refs/tags/v1.1.7^{}
9b65d57204c7837c82135cdf284e83dbc9ce55d6        refs/tags/v1.1.8
ba44d2cc33edbe88e842c46d33082be6ead5ca82        refs/tags/v1.1.8^{}
df404fb9a35408cd583bff5d15f8ad930e913f1d        refs/tags/v1.1.9
41707cdb00ce40c36c190bc4da9b42efee0e68ee        refs/tags/v1.1.9^{}
25a8843f1fd84ff779ded44348c53a87e16360d4        refs/tags/v2.0.0
91173c7289ad37914745c42e37573273885e21ec        refs/tags/v2.0.0^{}
826922059b685d57aa28d21ad8750e5dbabc9803        refs/tags/v2.0.1
d175d0f68745a5014fc9c41b25b8e0e959269126        refs/tags/v2.0.1^{}
4e27d172c8dc040d0ebe55625ae53b44f695bcc2        refs/tags/v2.0.2
1a70134a71196eeabb5e27bc7580faaa68d30513        refs/tags/v2.0.2^{}
bce5c83bba35234d7b98f7da4408096b5d5766ca        refs/tags/v2.0.3
58099551e1ea7d99f8d13495740c5ae3f5d98053        refs/tags/v2.0.3^{}
06576dcb258bfb18e978930fb387a4d1e65be0d9        refs/tags/v2.0.4
f2a4f4a659bacbe742a494b7d2aede64fab0d4f9        refs/tags/v2.0.4^{}
40e41dad21b120fe9d2c2aad8357d2f9547acf94        refs/tags/v2.0.5
a4f23d8e451267684a8160679ce16e16149fe72b        refs/tags/v2.0.5^{}
00c35781a640bce3717b130543109089edc49a01        refs/tags/v2.0.6
01fb5608a970b42c900b96746cadc13d27dd9d7e        refs/tags/v2.0.6^{}
95e15aac138f183e164034dc170bef5fc6b820d2        refs/tags/v2.1.0
4c2d8e2639ffd516b12544ce0c117cc0345daa3f        refs/tags/v2.1.0^{}
8985e6a889510414a22a0a086f1d25ec8075724b        refs/tags/v2.1.1
c3d049d11dc8a57af3135a9b8af3279316603a42        refs/tags/v2.1.1^{}
1945a603de3961d94d34bbc785ce83ed4d68ede6        refs/tags/v2.1.2
06754eed4f2b882b589f8667ecc8aadcf916045f        refs/tags/v2.1.2^{}
8ad92cd2f6e1a02f826b574e2d4f31e824f3f32e        refs/tags/v2.1.3
abcff84645534aaedaed4192c4ce788af9112bc2        refs/tags/v2.1.3^{}
f64ccc307387e8b76cc8a7dc1e9fe4b01b43c3b9        refs/tags/v2.1.4
7752832fba237929388dea6c96911a0a6379abfc        refs/tags/v2.1.4^{}
0c4ce079e9a944a16e05dcf2cb693b81d8293357        refs/tags/v2.1.5
758c8b3845af855fde736b6a7f58a15fba00d1e7        refs/tags/v2.1.5^{}
777bfdf5a860a96ea1b5acc0a8c7ce628e67bf43        refs/tags/v2.2.0
54a2d71a68fbd182854976675657f5a26d61d497        refs/tags/v2.2.0^{}
2803bfde1b234416104cd59d7a8889bfb704289b        refs/tags/v2.2.1
8f97e1d45b8e67cd37d8736ab4bb69053b7b18a3        refs/tags/v2.2.1^{}
fc20e297b1e0a2ea1c7b466be604a4e9bbb9a946        refs/tags/v2.2.10
2e9112d7d31a2af6e0682db0e18679b1e5fd4694        refs/tags/v2.2.10-meteor
98b5c7625364b302e418d0412429bc8d228d356a        refs/tags/v2.2.11
5cbab1210689b90cf660bd561de7abc4dec2dc72        refs/tags/v2.2.2
f239454e183d2032c0eb7d79a1c08f674fdd8db4        refs/tags/v2.2.2^{}
f322a75b3ea274f56fa8223e96525f42f6de72f4        refs/tags/v2.2.3
4dbe6e20051e44025d3cbec3ed9603483d8b27cb        refs/tags/v2.2.3^{}
ed72a423e4987fedbdcaafb00a995516c6a4abdc        refs/tags/v2.2.4
c271a6ab33645adc17eca9ad4592cfbe301c973a        refs/tags/v2.2.4^{}
b062fe03ee8b1a88afacd468310dbd11bf811ec0        refs/tags/v2.2.5
f94eebc1c5b637e6575735e6923ec79ee3139284        refs/tags/v2.2.6
847d79ea060d512ddd07f5e246c12a38520da308        refs/tags/v2.2.7
2390aec4ee4f53c0a33155b801f0c97cca5c8bb6        refs/tags/v2.2.8
645db649af1d7855ed52eab3e087e6b320000867        refs/tags/v2.2.9
b36754167183ef179c8a4a8bd336e82ce4aa5f70        refs/tags/v2.3.0
f81f716335c7c5f500618c7ee8fd29e08b1cf2b5        refs/tags/v2.3.0^{}
46770aba1db21c164ee0df8b576576102700236b        refs/tags/v2.3.1
dfbb3377d15aa42500c9fdedbcf3a2819204bf7b        refs/tags/v2.3.2
cd59995050105b946884ee20e3bcadc252feda8c        refs/tags/v2.3.3
64c3e968c4cc28fb42ad7e8a6c66bb970069b8e2        refs/tags/v2.3.3-meteor
```

**Comment by benjamn** [comment]
Is that the same `git` as `C:\Program Files\Git\cmd\git.EXE`? Sorry, I'm not entirely sure how to check that on Windows. Possibly you can type `where git` in `cmd.exe`?

**Comment by nathantreid** [comment]
Indeed, that is the path:
``` bat
C:\scratch\1.6rc15a>where git
C:\Program Files\Git\cmd\git.exe
```

**Comment by benjamn** [comment]
And `git version`? 2.14.2 is the latest Windows release fwiw.

**Comment by nathantreid** [comment]
```
C:\scratch\1.6rc15a>git version
git version 2.12.2.windows.2
```
I'll update git and see what happens.

**Comment by nathantreid** [comment]
That did it! Just installed `2.14.2.windows.3` and now `meteor create` and `meteor npm i` are working. Thanks a ton!

**Comment by benjamn** [comment]
Whew! 😌 It's good to know this is a possible solution if the same thing happens to other developers (and I'm afraid it will) after Meteor 1.6 comes out. I guess Git does a better job of shipping with SSL support baked in now.

As an aside, it's a bit disconcerting that `git` is now essentially a requirement of `npm`, at least since `npm` started using https://github.com/zkat/pacote to fetch packages.

**Comment by antoninadert** [comment]
I have version 2.13.0 of Git

I had this error twice already. It seems to happen somehow after I deleted my node_modules folder (or after discarding changes in my package.json).

Once it was because one package of my node_modules didn't get properly deleted, so I had to delete this one to run the command. 

The second time node_modules was clean and I didn't understand, so I did `npm install` without the meteor prefix.... 

then I did `meteor` 

and it worked (I can't say why)
